RMSD.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import os, sys, glob
+import openeye.oechem as oechem
+import openmoltools
+
+#-------------------- Functions---------------------#
+
+
+def RMSD(ffRef, ffList, molSet, molName):
+    """
+    From a set of reference molecules and query molecules, the RMSD values are
+     computed and wrote into a file.  
+   
+    Parameters
+    ---------
+    ffRef: str - the reference forcefield
+    ffList: str - the query forcefield
+    molSet: str - set of molecule
+    molName: str - name of the molecules
+   
+    Returns
+    -------
+    RMSD values for each molecules with the defined ff
+ 
+    """
+    # Set up an error file to record molecule does not exist
+    errFile = open('rmd_logfile.txt','a')
+
+    # Set up a file that catch negative values
+    nValue = open('negative_value.txt','a') 
+  
+    # Set up a log file for RMSD
+    logFile = open('RMSD.txt', 'a')
```

Comment:
These should be defined outside the function and passed in OR you could return the things you need to write to the output files. That way you could define those files to write to an output directory that can be more easily customized.